src/core/Form/RadioButton/RadioButton.baseStyles.tsx

```diff
@@ -71,11 +70,14 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       ${theme.typography.bodyTextSmall};
     }
     & .fi-radio-button_label {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
```

Comment:
Is this just for the vertical alignment? Is there a reason for using this method instead of, for example, setting the line height to match the element height? I think that one is used elsewhere.

Reply:
Yes, it seems it was just for aligning the label vertically.

This is now changed to use `line-height` instead; same for the large variant.